src/cogs/trivia.py

```diff
@@ -6,6 +6,8 @@
 
 locked_channels = []  # Channel IDs that are currently in use by a game
 
+questions = []  # Questions that have been sent
```

Comment:
Given this scope, won't this affect *all* games?

Reply:
Fixed